Scripts/Common/Deblob.sh

```diff
@@ -648,14 +650,15 @@ echo "Deblobbing...";
 	manifests=$manifests"|wfdhdcp|wifidisplayhdcphal|WifiDisplay";
 
 	#Widevine (DRM) [Google]
-	blobs=$blobs"|libdrmwvmplugin.so|libmarlincdmplugin.so|libwvdrmengine.so|libwvdrm_L1.so|libwvdrm_L3.so|libwvhidl.so|libwvm.so|libWVphoneAPI.so|libWVStreamControlAPI_L1.so|libWVStreamControlAPI_L3.so|libdrmmtkutil.so|libsepdrm.*.so|libvtswidevine32.so|libvtswidevine64.so|android.hardware.drm.*widevine.*";
-	blobs=$blobs"|test-wvdrmplugin|oemwvtest";
-	blobs=$blobs"|com.google.widevine.software.drm.jar";
-	blobs=$blobs"|com.google.widevine.software.drm.xml";
-	#blobs=$blobs"|smc_pa_wvdrm.ift"; breaks maguro/toro* boot
-	blobs=$blobs"|tzwidevine.*|tzwvcpybuf.*|widevine.*";
-	makes=$makes"|libshim_wvm|move_widevine_data.sh|android.hardware.drm.*widevine.*";
-
+    if [ "$DOS_DEBLOBBER_REMOVE_WIDEVINE_DRM" = true ]; then
+		blobs=$blobs"|libdrmwvmplugin.so|libmarlincdmplugin.so|libwvdrmengine.so|libwvdrm_L1.so|libwvdrm_L3.so|libwvhidl.so|libwvm.so|libWVphoneAPI.so|libWVStreamControlAPI_L1.so|libWVStreamControlAPI_L3.so|libdrmmtkutil.so|libsepdrm.*.so|libvtswidevine32.so|libvtswidevine64.so|android.hardware.drm.*widevine.*";
+		blobs=$blobs"|test-wvdrmplugin|oemwvtest";
+		blobs=$blobs"|com.google.widevine.software.drm.jar";
+		blobs=$blobs"|com.google.widevine.software.drm.xml";
+		#blobs=$blobs"|smc_pa_wvdrm.ift"; breaks maguro/toro* boot
+		blobs=$blobs"|tzwidevine.*|tzwvcpybuf.*|widevine.*";
+		makes=$makes"|libshim_wvm|move_widevine_data.sh|android.hardware.drm.*widevine.*";
+    fi;
```

Comment:
.. and this this should be indented

Reply:
i don't get it. what should be indented? the "fi;"?